### DISCUSSION TITLE
Merge Issues in a centralized place?

### DISCUSSION THREAD

**Discussion by shinebayar-g** [cdk8s-team/cdk8s#1274] [General]
cdk8s has many repositories. For easier visibility and usability, could we close **Issues** tab on sub repositories and manage them here? Because from the end user perspective it's just **cdk8s** and it's not clear to which repository they should check and search for issues or submit them. By default, they come here, but there are some good discussions going on sub repos. It feels like things are all over the place between these repos.
https://github.com/cdk8s-team/cdk8s/issues
https://github.com/cdk8s-team/cdk8s-core/issues
https://github.com/cdk8s-team/cdk8s-cli/issues
Maintainers could apply appropriate labels to distinguish them when necessary.

**Comment by iliapolo** [comment]
@shinebayar-g apologies for the long delay in responding to this. But I'm actually open to the idea and wanted to bring it up for discussion with the team and in our slack channel. 

Are you still interested in this?

**Reply by shinebayar-g** [reply]
No worries, it's just a suggestion. I think it would make yours/maintainers' life easier, win/win for everyone.

**Comment by iliapolo** [comment]
I actually like the idea. Lets see what folks think of this. I'll reach out on slack.

**Reply by shinebayar-g** [reply]
Hey @iliapolo , I remember the poll was voted "yes" for the majority. Could you please reconsider this suggestion?
You could tag issues with the appropriate labels in the cdk8s-team/cdk8s repo like `cli` or `component:cli` etc..
It's getting tricky to keep track of all the reported issues across multiple repositories.

**Comment by iliapolo** [comment]
@shinebayar-g Done. 

<img width="1163" alt="Screenshot 2024-09-22 at 15 10 10" src="https://github.com/user-attachments/assets/77923e44-6c49-4bd7-b76e-ac8b087c5554">

**Reply by shinebayar-g** [reply]
Awesome. That's very nice you were able to link to the https://github.com/cdk8s-team/cdk8s/issues/new/choose

I noticed that [GitHub Community Support](https://github.com/cdk8s-team/cdk8s-core/discussions) option could be linked to the cdk8s-team/cdk8s repository as well. What do you think?

![image](https://github.com/user-attachments/assets/3fe54c3e-1909-4992-a30e-5466cc079111)
